frontend/src/app/core/setup/globals/onboarding/helpers.ts

```diff
@@ -2,6 +2,7 @@ export const demoProjectName = 'Demo project';
 export const scrumDemoProjectName = 'Scrum project';
 export const onboardingTourStorageKey = 'openProject-onboardingTour';
 export type OnboardingTourNames = 'prepareBacklogs'|'backlogs'|'taskboard'|'homescreen'|'main';
+export type ProjectName = 'demo'|'scrum';
```

Comment:
You could make this an enum, but it's not super important

Reply:
I was too slow for Oliver ;) I changed it here: https://github.com/opf/openproject/pull/10369